docs/conf.py

```
@@ -46,6 +46,8 @@
 sys.path.insert(0, os.path.abspath(".."))
 from docs.utils import _validate_git_ref  # noqa
 from docs.utils import slugify_header  # noqa
+from docs.utils import sanitize_url  # noqa
```

Ruff (0.8.2)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)